src/Generator/Commands/ActionGenerator.php

```diff
@@ -76,20 +76,99 @@ protected function getFilePath(): string
 
     protected function getStubFileName(): string
```

Comment:
This method needs another name, like `GenerateFileContent` or something. This is true for most of the methods, we should not refer to any stub, if we dont use it anymore.

Reply:
Done.